website/content/docs/internals/plugins/task-drivers.mdx

```diff
@@ -96,6 +92,30 @@ The network isolation modes are:
 - `NetIsolationModeNone`: There is no network to isolate. This is used for
   task that the client manages remotely.
 
+#### Remote Task Drivers
+
+[Remote Task Drivers][rtd] should set `RemoteTasks` to `true`. Remote Task
+Drivers are task driver plugins that execute tasks on a different system than
+the Nomad client. This means the tasks lifecycle is distinct from the Nomad
+client's.
```

Comment:
maybe "from the Nomad client node"? 

It's always been unclear to me when "Nomad client" == agent or node, and here the distinction is important

Reply:
> It's always been unclear to me when "Nomad client" == agent or node, and here the distinction is important

Same. `client` seems to be used pretty broadly on this page at least. `node` appears once and `agent` does not appear.

So I think I'll stick with the status quo for now and punt on providing clarity. :grimacing: